turbinia/workers/workers_test.py

```diff
@@ -150,6 +151,16 @@ def testTurbiniaTaskRunWrapperBadResult(self):
     self.assertEqual(type(new_result), TurbiniaTaskResult)
     self.assertIn('CheckedResult', new_result.status)
 
+  def testTurbiniaTaskJobUnavailable(self):
+    """Test that the run wrapper can fail if the job doesn't exist."""
+    self.setResults()
+    self.task.job_name = 'non_exist'
+    canary_status = 'Task will not run due to the job: '\
```

Comment:
I would wrap this string in parens rather than use a backslash to escape the newline.  It mentions something about this in the style guide here: http://google.github.io/styleguide/pyguide.html#32-line-length